### DISCUSSION TITLE
在线蹲一个补丁教程

### DISCUSSION THREAD

**Discussion by awaae001** [Melledy/Grasscutter#2299] [Support]
版本更新之后，补丁似乎都要更新。
每次等待更新的时间总是很漫长的。
蹲一个自编译

**Comment by Pawsedawa** [comment]
[version.zip](https://github.com/Grasscutters/Grasscutter/files/12391328/version.zip)
或许你可以试试 我不确定是否能用 放在Yuanshen.exe 同目录下就行了
